verify-replace.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+TMP_DIR=/tmp/
+BASE_REPO_PATH=$(mktemp -d ${TMP_DIR}replace-verify.XXX)
+GH_BASE_URL_KS=https://github.com/kubesaw/
+GH_BASE_URL_CRT=https://github.com/codeready-toolchain/
+GH_KSCTL=${GH_BASE_URL_KS}ksctl
+GH_HOST=${GH_BASE_URL_CRT}host-operator
+GH_MEMBER=${GH_BASE_URL_CRT}member-operator
+GH_REGSVC=${GH_BASE_URL_CRT}registration-service
+GH_E2E=${GH_BASE_URL_CRT}toolchain-e2e
+GH_TC=${GH_BASE_URL_CRT}toolchain-common
```

Comment:
I'm also not sure if toolchain-common should be here. This would download the repo in tmp and validate it which is not needed in this case right ?

Reply:
removed it